pkg/basecluster/data.go

```diff
@@ -67,8 +67,6 @@ nameReference:
   fieldSpecs:
   - path: spec/bootstrap/configRef/name
     kind: Machine
-  - path: spec/template/spec/bootstrap/configRef/name
```

Comment:
Why is this being removed? Was it incorrect?

Reply:
Nope.....I will add it back here.....Just need to remove the template and make some changes in the setup script